### PR TITLE
complete 9175

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description

after create operator role for Gabriele D annunzio check delegation menu side item
## List of changes proposed in this pull request

- insert check for delegation side menu item

## How to test

./mvnw clean test -Dcucumber.filter.tags=@TA_PGRuoloOperatoreControlloAssenzaDeleghe

- [ ] Does this PR require a manual test in AWS CodeBuild?

### CODE REVIEW

File: src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/HomePagePG.java [hunk 1]
```diff
@@ -21,26 +18,51 @@ public HomePagePG(WebDriver driver) {
...
+            isDisplayed = false; // Elemento non trovato
+        }
+
+        if (isDisplayed) {
+            logger.info("Il side menu deleghe è visibile");
+        } else {
+            logger.info("Il side menu deleghe non è presente");
+        }
+
+        Assert.assertTrue("Il side menu deleghe non è visibile", !isDisplayed);
```
Comment (azhansultan) [inline src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/HomePagePG.java:57]: does if/else needed ? line 57-61

Comment (leleOFA) [reply src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/HomePagePG.java:57]: done

File: src/test/resources/test-config.properties [hunk 2]
```diff
@@ -2,7 +2,7 @@
 browser=chrome
 environment=test
 cookie.config=true
-headless=true
+headless=false
```
Comment (azhansultan) [inline src/test/resources/test-config.properties]: change to true

**Review by azhansultan — APPROVED** [review]

File: src/test/resources/test-config.properties (hunk 2 above)
Comment (leleOFA) [reply src/test/resources/test-config.properties]: done

**Pull request merged**